core-bundle/contao/drivers/DC_File.php

```diff
@@ -329,22 +329,11 @@ public function edit()
 			$this->reload();
 		}
 
-		// Set the focus if there is an error
-		if ($this->noReload)
-		{
-			$return .= '
-<script>
-  window.addEvent(\'domready\', function() {
-    Backend.vScrollTo(($(\'' . $this->strTable . '\').getElement(\'label.error\').getPosition().y - 20));
-  });
-</script>';
-		}
-
 		$return = '
-<div data-controller="contao--jump-targets">
+<turbo-frame id="tl_edit_form_frame" target="_top" data-turbo-action="forward" data-controller="contao--jump-targets">
```

Comment:
Did you mean `data-turbo-action="advance"`?

Reply:
Yes 🙈